lib/offensive.rb

```diff
@@ -1,15 +1,22 @@
 module Offensive
 
-  def offensive
-    offense = Hash.new {|hash, key| hash[key] = []}
-    @games.map do |game|
-      offense[game.away_team_id] << game.away_goals
-      offense[game.home_team_id] << game.home_goals
-    end
-    average = {}
-    offense.each do |id, goals|
-      average[id] = goals.sum / goals.count.to_f
-    end
-    average
+  def offensive(*away_or_home)
```

Comment:
this is such a small detail and I don't think it matters but is the asterisk in '*away_or_home' on purpose?